### PR TITLE
remove older versions of elemental

### PR DESCRIPTION
**Pull request by aalves08**
remove older versions of elemental

@jordojordo , will this sync automatically? meaning, will it remove these versions and generate the chart again?

### CODE REVIEW

**Comment by jordojordo** [comment]
@aalves08 This will not remove the versions that have already been synced. But I'm curious why you would want to remove the older versions?

**Comment by aalves08** [comment]
It was a request from the backend developer I've talked with. Apparently it creates more havoc because they have to support multiple versions of the images for the OS iso build. They used to be stored in other places and now everything should be centralised like version `1.2.0`.

FYI @nwmac

**Comment by jordojordo** [comment]
Makes sense, in that case you will need to remove all of the chart assets for those versions in `assets`, `charts`, and `extensions` - In addition the entries within for those versions will need to be removed from the `index.yaml`.

This will not remove the versions from the [releases](https://github.com/rancher/ui-plugin-charts/releases) or the [`ui-plugin-catalog` image](https://hub.docker.com/r/rancher/ui-plugin-catalog/tags) unfortunately, but we can release a patch version after this is merged which will not contain those versions.

**Comment by aalves08** [comment]
@jordojordo can you re-review this? I've removed all of the assets needed to do a proper cleanup.

File: assets/elemental/elemental-1.2.0.tgz
Comment (jordojordo) [inline assets/elemental/elemental-1.2.0.tgz:1]: Should this be deleted?

**Review by jordojordo — CHANGES_REQUESTED** [review]

File: assets/elemental/elemental-1.2.0.tgz
Comment (aalves08) [reply assets/elemental/elemental-1.2.0.tgz:1]: No it shouldn't...

**Review by jordojordo — APPROVED** [review]

**Pull request merged**